scheduler/src/cook/mesos/api.clj

```diff
@@ -975,6 +1025,25 @@
      :exists? (partial retrieve-jobs conn)
      :handle-ok (fn [ctx] (render-jobs-for-response conn framework-id retrieve-sandbox-directory-from-agent ctx))}))
 
+(defn read-jobs-handler
+  [conn framework-id is-authorized-fn retrieve-sandbox-directory-from-agent handle-ok-fn]
+  (base-cook-handler
+    {:allowed-methods [:get]
+     :malformed? job-request-malformed?
+     :allowed? (partial job-request-allowed? conn is-authorized-fn)
+     :exists? (partial jobs-exist? conn)
+     :handle-ok handle-ok-fn}))
+
+(defn read-jobs-handler-multiple
+  [conn framework-id is-authorized-fn retrieve-sandbox-directory-from-agent]
+  (read-jobs-handler conn framework-id is-authorized-fn retrieve-sandbox-directory-from-agent
+                     (partial render-jobs-for-response conn framework-id retrieve-sandbox-directory-from-agent)))
+
+(defn read-jobs-handler-single
+  [conn framework-id is-authorized-fn retrieve-sandbox-directory-from-agent]
+  (let [handle-ok-fn (fn [ctx]
+                       (first (render-jobs-for-response conn framework-id retrieve-sandbox-directory-from-agent ctx)))]
+    (read-jobs-handler conn framework-id is-authorized-fn retrieve-sandbox-directory-from-agent handle-ok-fn)))
```

Comment:
Nitpick: `->>` would help with readability and help us avoid the `let`

Reply:
I think refactoring `read-jobs-handler` (as discussed above) would fix this too.

Reply:
See other comment about the same topic